lib/statsd.rb

```diff
@@ -329,16 +320,15 @@ def format_event(title, text, opts={})
   private
 
   def escape_event_content(msg)
-    msg.gsub! "\n", "\\n"
+    msg.gsub "\n", "\\n"
   end
 
-  def rm_pipes(msg)
-    msg.gsub! "|", ""
+  def remove_pipes(msg)
```

Comment:
I had renamed this method to `without_pipes`, since it doesn't remove anything from the passed String, but since @yannmh prefers imperative verbs, I decided to rename it to `remove_pipes`. (I know this is not the original method name, but personally I consequently avoid abbreviations in method names. @yannmh: If you disagree with this change, just tell me and I'll change it back to the original method name.)